### DISCUSSION TITLE
One vs all comparisons

### DISCUSSION THREAD

**Discussion by jindalk** [GreenleafLab/ArchR#1554] [Questions / Documentation] [answered]
Within a specific subset of clusters in my ArchR object, is it possible to perform one vs all comparison for each of them? For example, for 5 of my 10 clusters, can I compare each one of the 5 clusters to the rest 4, in a given feature space? I am not sure how to use `getMarkerFeatures` to force this specific behaviour.

**Comment by rcorces** [comment, marked as answer]
I think you could accomplish what you want in two ways:
1) Creating a new column in cellColData that explicitly maps your one-vs-all comparison
2) Setting your one group as useGroups and the all as bgdGroups
